src/consumption.cpp

```diff
@@ -14,10 +14,13 @@
 #include "mutation.h"
 #include "options.h"
 #include "translations.h"
+#include "units.h"
 
 #include <string>
 #include <algorithm>
 
+using namespace units::literals;
```

Comment:
If namespacing requires a using statement in order to use the literal designator, the literals should be moved out of the units namespace.  It's pointless to declare something in a namespace and then require the name space to be used to call it.